lib/WooCommerce/Helper.php

```diff
@@ -35,6 +35,10 @@ public function getWoocommerceCurrency() {
     return get_woocommerce_currency();
   }
 
+  public function getWoocommerceCurrencySymbol() {
+    return get_woocommerce_currency_symbol();
```

Comment:
What happens when the store uses multi currencies?

Reply:
Good question. I will find out how multiple currencies work and let you know.

Reply:
I don't think WooCommerce supports multiple currencies by default, you need a plugin for that (and there is plenty of them), so I'm not sure if we want to support each plugin specifically.

Reply:
True. There is one on our marketplace: https://woocommerce.com/products/multi-currency/# 

We don't need to support multiple currencies. But at least we should know what happens and handle that gracefully.

Reply:
I have tested the multi-currency plugin, and everything works well because the plugin recalculates prices for products on the page shop with the rate to the main currency. It means that this function returns the main currency, which is configured in WooCommerce.